src/java/com/rapleaf/jack/queries/Functions.java

```diff
@@ -0,0 +1,54 @@
+package com.rapleaf.jack.queries;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Collection;
+import java.util.List;
+
+import com.google.common.collect.Lists;
+
+public class Functions {
+
+  private static final DateFormat datetimeFormat = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss");
```

Comment:
SimpleDateFormat is not threadsafe -- should use DateTimeFormatter from Joda time.